.env.development

```diff
@@ -0,0 +1,8 @@
+HANDLE_TARGET_HOSTNAME=localhost:3000
+HANDLE_EMAIL=contact@example.com
+HANDLE_PREFIX=1234.DEV
+HANDLE_ADMIN_PREFIX='"1234.TEST/ADMIN"'
```

Comment:
The double quote convention seems really really fragile.  Is this really required - I see it for HANDLE_EMAIL in the test version, but not here.  There's no-way I would guess from the sample that I needed '"value"'

Reply:
If the values aren't valid YAML, the quoting is required. Otherwise when these values are interpolated into the YAML string, we get parser errors.

I agree that this is cumbersome and it has already cost me at least an hour of frustrated debugging, between various projects.